### PR TITLE
QPPCT-428: prevent ecqm duplicates

### PR DESCRIPTION
**Pull request by clydet**
### Information
- QPPCT-428

### Changes proposed in this PR:
- Prevents duplicate quality measures

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 1]
```diff
@@ -8,15 +8,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.EnumMap;
 import java.util.EnumSet;
+import java.util.HashSet;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'java.util.HashSet'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 2]
```diff
@@ -8,15 +8,22 @@
...
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+import java.util.function.Predicate;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'java.util.function.Predicate'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 3]
```diff
@@ -8,15 +8,22 @@
...
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'java.util.stream.Stream'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 4]
```diff
@@ -8,15 +8,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'java.util.ArrayList'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 5]
```diff
@@ -8,15 +8,22 @@
...
 import java.util.Arrays;
+import java.util.Collections;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'java.util.Collections'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 5 issues
* ![MINOR][MINOR] 5 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by saquino0827** [comment]
I forgot to add that you may need to modify the `ERROR_MESSAGES.md` file to add your new error message

**Pull request merged**